test_scripts/LowVoltage/006_App_unregister_itself_gracefully.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------------
+--
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/LowVoltage/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hashId = { }
+
+--[[ Local Functions ]]
+local function addResumptionData()
+  local pAppId = 1
+  local cid = common.getMobileSession(pAppId):SendRPC("AddCommand", { cmdID = 1, vrCommands = { "OnlyVRCommand" }})
+  common.getHMIConnection():ExpectRequest("VR.AddCommand")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession(pAppId):ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  common.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      hashId[pAppId] = data.payload.hashID
+    end)
+end
+
+local function checkResumptionData()
+  common.getHMIConnection():ExpectRequest("VR.AddCommand")
+  :Times(0)
+  common.getHMIConnection():ExpectRequest("BasicCommunication.UpdateAppList")
```

Comment:
@dboltovskyi , for what UpdateAppList is expected?

Reply:
@GetmanetsIrina ```BC.UpdateAppList``` expectation is removed since it's always sent regardless of app's status (resumed/not resumed)

Reply:
@GetmanetsIrina Done in 00e3718